core/src/main/java/io/kestra/plugin/core/http/Request.java

```diff
@@ -185,6 +215,59 @@
                         name: "my-file.txt"
                         content: "{{ inputs.file }}"
                 """
+        ),
+        @Example(
+            title = "Upload an image using HTTP POST request to a webserver.",
+            full = true,
+            code = """
+                id: http_upload_image
+                namespace: company.team
+                
+                tasks:
+                  - id: s3_download
+                    type: io.kestra.plugin.aws.s3.Download
+                    accessKeyId: "{{ secret('AWS_ACCESS_KEY_ID')}}"
+                    secretKeyId: "{{ secret('AWS_SECRET_KEY_ID')}}"
+                    region: "eu-central-1"
+                    bucket: "my-bucket"
+                    key: "path/to/file/my_image.jpeg"
+                  
+                  - id: send_data
+                    type: io.kestra.plugin.core.http.Request
+                    uri: "https://server.com/upload"
+                    headers:
+                      user-agent: "kestra-io"
+                    method: "POST"
+                    contentType: "image/jpeg"
+                    formData:
+                      user:
+                        file: "my-image.jpeg"
+                        url: "{{ outputs.s3_download.uri }}"
+                        metadata:
+                          description: "my favorite image"
+                """
+        ),
+        @Example(
+            title = "Upload a CSV file using HTTP POST request to a webserver.",
+            full = true,
+            code = """
+                id: http_csv_file_upload
+                namespace: company.team
+                
+                tasks:
+                  - id: http_download
+                    type: io.kestra.plugin.core.http.Download
+                    uri: https://huggingface.co/datasets/kestra/datasets/raw/main/csv/orders.csv
+                  
+                  - id: upload
+                    type: io.kestra.plugin.core.http.Request
```

Comment:
This is missing a URI

```suggestion
                    type: io.kestra.plugin.core.http.Request
                    uri: "https://server.com/upload"
```